src/components/ViewComments.jsx

```diff
@@ -68,13 +96,19 @@ const ViewComments = () => {
                 ></textarea> <br />
                 <button>Post comment</button>
                 
-                </form> <br />
+                </form> <br /> <br />
 
             {comments.map((comment) => <article key={comment.comment_id}>
+                
                 <strong>{comment.author}</strong> says...
-                 <div className="comments_body">{comment.body}</div>
-                 <button>💓 {comment.votes}</button> <br /> <br />
+                 <div className="comments_body">
+                    {comment.body} <br />
+                    {comment.author === user.username ? <button onClick={(e) => noComment(comment.comment_id)}>Delete</button> : null}
```

Comment:
This is a good way of determining if the delete button should show for each comment.

Reply:
I love the self-compliment here 😃